src/main/scala/nl.knaw.dans.easy.deposit/docs/DatasetMetadata.scala

```diff
@@ -93,7 +93,26 @@ case class DatasetMetadata(private val identifiers: Option[Seq[SchemedValue]] =
 
   /** Validations as far as not covered by DDM schema validation. */
   private[docs] def validate(): Try[Unit] = {
-    Author.validate(authors)
+    def missingMandatory[T <: Mandatory](msg: String, values: Seq[T]): Try[Unit] = {
+      val invalid = values
+        .withFilter(!_.hasMandatory)
+        .map(JsonUtil.toJson)
+        .mkString(", ")
+      if (invalid.isEmpty) Success(())
+      else {
+        Failure(new IllegalArgumentException(
+          s"Missing values for $msg: $invalid"
+        ))
+      }
+    }
+
+    for { // TODO collect errors over multiple fields, not just within the list of a field
+      _ <- Author.validate(authors)
+      // filterNot: stick to schema validation for now, error message of missingMandatory not clear
```

Comment:
Perhaps just ignore Contributors/Creators and other fields without any value. Opinions?

For that matter perhaps refactor use of `getNonEmpty`-s in `DDM.toXml`